.github/renovate.json5

```diff
@@ -1,12 +1,31 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
-  semanticCommits: "auto",
+  semanticCommits: "enabled",
+  schedule: [
+    "after 8am on sunday"
+  ],
+  prConcurrentLimit: 0,
+  prHourlyLimit: 0,
```

Comment:
Sends all the emails on Sunday. Hopefully, this should also limit PR updating to Sundays.

Reply:
The emails aren't send by renovate. They are run by GitHub when a PR is opened or changed. The schedule is for actually processing the repository and creating new PRs.